epoxy-adapter/src/main/java/com/airbnb/epoxy/EpoxyViewHolder.java

```diff
@@ -34,7 +34,7 @@ public void bind(@SuppressWarnings("rawtypes") EpoxyModel model,
 
     if (previouslyBoundModel != null) {
       // noinspection unchecked
-      model.bind(objectToBind(), previouslyBoundModel);
+      model.bindChanges(objectToBind(), previouslyBoundModel);
```

Comment:
yep, sorry if I'm getting too nitpicky with comments at this point since I know it is unfinished. Just wasn't sure what you were planning on changing, and what was a bug you didn't notice, so decided to err on letting you know :P